trade-finance-manager-api/src/v1/helpers/amendment.helpers.js

```diff
@@ -311,10 +311,34 @@ const sendFirstTaskEmail = async (taskVariables) => {
   }
 };
 
+/**
+ * Initiate an internal UKEF team email, upon an amendment
+ * submission (automatic) and approval (manual)
+ * @param {String} ukefFacilityId UKEF Facility ID
+ */
+const internalAmendmentEmail = async (ukefFacilityId) => {
+  try {
+    const templateId = EMAIL_TEMPLATE_IDS.INTERNAL_AMENDMENT_NOTIFICATION;
+    const sendToEmailAddress = process.env.UKEF_INTERNAL_NOTIFICATION;
+    const emailVariables = {
+      ukefFacilityId,
+    };
+
+    if (!ukefFacilityId) {
+      return false;
+    }
+
+    return sendTfmEmail(templateId, sendToEmailAddress, emailVariables);
```

Comment:
SonarCloud is flagging this as a potential bug because it needs the `await` part. is eslint saying something else? are you able to have a look?

Reply:
Yep `eslint` doesn't like `await` and woud like `return`.
I had `await` before.